benchmark/benchmark_test.go

```diff
@@ -69,7 +67,8 @@ var _ = Describe("Benchmarks", func() {
 
 				ln.Close()
 				sess.Close(nil)
-			}, 6)
+				}, samples)
```

Comment:
The file is properly gofmted. Please note that all `Describe` code was moved into the `init`. This is the only way to use the flag values.